shared/main/java/net/ccbluex/liquidbounce/utils/login/UserUtils.kt

```diff
@@ -40,14 +40,14 @@ object UserUtils {
         val request = HttpPost("https://authserver.mojang.com/validate")
         request.setHeaders(headers)
 
-        val body = JSONObject()
-        body.put("accessToken", token)
-        request.entity = StringEntity(body.toString())
+        val body = JsonObject()
+        body.addProperty("accessToken", token)
+        request.entity = StringEntity(Gson().toJson(body))
+        println(Gson().toJson(body));
```

Comment:
there is a debug print out still left in the code

Reply:
emmm i forgot to delete my debug code